---
layout: reference
section: learn
title: Exception
permalink: /learn/ref/Microsoft.Coyote.Tasks/Task/Exception
---
# Task.Exception property

Gets the AggregateException that caused the task to end prematurely. If the task completed successfully or has not yet thrown any exceptions, this will return null.

```csharp
public AggregateException Exception { get; }
```

## See Also

* class [Task](../TaskType)
* namespace [Microsoft.Coyote.Tasks](../TaskType)
* assembly [Microsoft.Coyote](../../MicrosoftCoyoteAssembly)

<!-- DO NOT EDIT: generated by xmldocmd for Microsoft.Coyote.dll -->
